### PR TITLE
APU

### PR DESCRIPTION
**Pull request by laanwj**
Info from: `lib/drivers/apu.c` `lib/drivers/include/apu.h`

### CODE REVIEW

File: k210.svd [hunk 1]
```diff
@@ -2556,11 +2556,352 @@
...
+                    <addressOffset>0x134</addressOffset>
+                    <fields>
+                        <field>
+                            <name>upper</name>
+                            <description>Upper limit</description>
+                            <bitRange>[15:0]</bitRange>
+                        </field>
+                        <field>
+                            <name>lower</name>
+                            <description>Lower limit</description>
```
Comment (Disasm) [inline k210.svd]: Maybe this one is `bottom`, but I'm not qualified in this area.

Comment (laanwj) [reply k210.svd]: Same—I don't really know what the saturation limits do here, it's not described in the API PDF, it's also not used in any of the examples.

Comment (laanwj) [reply k210.svd]: Oh you mean the parameter is called `bottom` in `apu_voc_set_saturation_limit`, sure, can change it to that.

Comment (Disasm) [reply k210.svd]: Yes, it's called `bottom` in `apu.c` file.
P.S.: It would be better if you fix things in separate commit so I can review changes quickly

Comment (laanwj) [reply k210.svd]: Renamed in an added commit. I'm used to projects that prefer whole commits, that's why I squashed every time.

**Review by Disasm — APPROVED** [review]
Thanks!

**Pull request merged**